### PR TITLE
priority system for events page, food coupon system, 

### PR DESCRIPTION
**Pull request by vsharma-va**

### CODE REVIEW

**Comment by notarib-catcher** [comment]
@k-srivastava

File: src/routes/api/vendor/scan/+server.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		vendor_auth_key: vendorAuthKey,
+		suspended: false
+	});
+
+	if (!couponCode) {
+		return json({ status: '400', detail: 'please enter a valid code' });
+	}
+
+	if (foundVendor != null) {
+		console.log(couponCode);
```
[the remark this answers is not in the document]
Comment (vsharma-va) [reply src/routes/api/vendor/scan/+server.js:38]: Sorry I forgot to change somethings back. Will fix and add another commit.

**Comment by vsharma-va** [comment]
Commented out all console.log statements and reverted secret to process.env.SECRET

**Review by notarib-catcher — APPROVED** [review]
Works

**Pull request merged**